.github/workflows/release.yaml

```diff
@@ -13,8 +13,20 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
+          
+      - name: Set tag output
+        id: tag
+        run: echo "tag_name=${GITHUB_REF#refs/*/}" >> "$GITHUB_OUTPUT"
+
+      - name: Install tejolote
+        uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
+
+      - run: |
```

Comment:
nit: To avoid script injection, it's often safer to store these values that can be user-modified (a tag can be user-modified) in an environment variable like

```
env:
  TAG: ${{ github.ref_name }}
run: |
  ...
```